lib/formatters/__tests__/stringFormatter.test.mjs

```diff
@@ -158,10 +443,10 @@ path/to/file.css
 
 		expect(output).toBe(stripIndent`
 path/to/file.css
- 1:1  ×  Unexpected very very very very very very very  bar-very-very-very-very-very-long
-         very very very very very very long foo
+  1:1  ×  Unexpected very very very very very very very  bar-very-very-very-very-very-long
```

Comment:
minor nit

I would have expected this one to have required one character to be removed to pass.

Reply:
Huh, yeah. I dug into this and it's an off-by-one error in the existing implementation. On `main`, this line is only 89 characters long. Adding the one extra space in this branch pushes it up to 90.

The cause of the error is the hardcoded `MARGIN_WIDTHS` value: https://github.com/stylelint/stylelint/blob/c49f9b5a7b45cb35c58b5bc1cd669733896d0183/lib/formatters/stringFormatter.mjs#L17

By default, `table` adds one character of padding to each side of each column. So a five column table would have 10 characters of padding. This can be overridden in the table config by setting paddingRight and paddingLeft. For `main`, that's done like this: https://github.com/stylelint/stylelint/blob/c49f9b5a7b45cb35c58b5bc1cd669733896d0183/lib/formatters/stringFormatter.mjs#L213-L221

So this table only had 8 characters of padding, but `MARGIN_WIDTHS` was set to 9. Which resulted in the table calculations coming out one character short.

This PR modifies the table config, bringing the actual padding value up to 9:  https://github.com/stylelint/stylelint/blob/ef5133519b8fe989ce92e678b6c683eefa122bfa/lib/formatters/stringFormatter.mjs#L213-L221

I've added a test to ensure the formatter will use up to the maximum available space.

A real fix would be to derive the margin widths from the table config itself, instead of using the `MARGIN_WIDTHS` constant. But I think that's outside the scope of this PR.

Reply:
> it's an off-by-one error

So my hunch was correct.

> I've added a test to ensure the formatter will use up to the maximum available space.

You have gone above and beyond of what I was expecting.